src/schedule/demand_schedule_strategy.py

```diff
@@ -112,7 +112,7 @@ def _compute_trains_to_spawn(self, produced_electrical_power: float) -> float:
         coal_consumption = self.compute_coal_consumption(produced_electrical_power)
         return coal_consumption / self.COAL_PER_TRAIN
 
-    def _calculate_spawn_ticks(self):
+    def calculate_spawn_ticks(self):
```

Comment:
This is called during construction in `__init__`. There is no need to call from the outside. Therefore you don't have to make in public.

Reply:
Got it :) Will change it back